.github/workflows/ci.yml

```diff
@@ -21,6 +21,23 @@ jobs:
     - name: run prometheus exporter tests
       run: ./ci/do_ci.sh cmake.exporter.prometheus.test
 
+  cmake_gcc_48_test:
+    name: Legacy CMake
```

Comment:
How about "CMake gcc 4.8" so it's clear that isn't not a legacy version of CMake that's being tested?

Reply:
Done